src/terminal/osc.zig

```diff
@@ -1147,6 +1170,22 @@ pub const Parser = struct {
         self.temp_state.str.* = self.buf[self.buf_start..self.buf_idx];
     }
 
+    fn endConEmuSleepValue(self: *Parser) void {
+        switch (self.command) {
+            .sleep => |*v| {
+                const str = self.buf[self.buf_start..self.buf_idx];
+                if (str.len != 0) {
+                    if (std.fmt.parseUnsigned(u16, str, 10)) |num| {
+                        v.* = @min(num, 10000);
+                    } else |_| {
+                        v.* = 10000;
```

Comment:
Not sure if we need to replicate ConEmu's behaviour here.

Reply:
What's ConEmu's behavior? We should either replicate ConEmu's behavior or document the differences.

Reply:
https://github.com/Maximus5/ConEmu/blob/740b09c363cb16fbb730d72c53eaca1c530a016e/src/ConEmuCD/ConAnsiImpl.cpp#L705-L724

Reply:
OK, looks like we match ConEmu's behavior. Please be careful linking to code though as we can't use GPL code since we are MIT. ConEmu is BSD 3 clause so I think that's OK (IANAL).